source/features/comments-time-machine-links.tsx

```diff
@@ -119,10 +119,10 @@ async function showTimemachineBar(): Promise<void | false> {
 
 function init(): void {
 	// PR reviews' main content has nested `.timeline-comment`, but the deepest one doesn't have `relative-time`. These are filtered out with `:not([id^="pullrequestreview"])`
-	const comments = select.all(`
-		:not(.js-new-comment-form):not([id^="pullrequestreview"]) > .timeline-comment:not(.rgh-time-machine-links),
-		.review-comment > .previewable-edit:not(.is-pending):not(.rgh-time-machine-links)
-	`);
+	const comments = select.all([
+		':not(.js-new-comment-form):not([id^="pullrequestreview"]) > .timeline-comment:not(.rgh-time-machine-links)',
+		'.review-comment > .previewable-edit:not(.is-pending):not(.rgh-time-machine-links)'
+	]);
```

Comment:
Does this need to be commented? Can you add comments if they help?

Reply:
Comments wont help much. I think the classes are self explanatory

Reply:
Good. I'll revert